scripts/build_arbor.sh

```diff
@@ -1,5 +1,6 @@
 arb_repo_path=$ns_build_path/arbor
 arb_build_path=$arb_repo_path/build
+modcc_build_path=$arb_repo_path/build.modcc
```

Comment:
An underscore would be a bit clearer, i.e `modcc_build_path=$arb_repo_path/build_modcc`

Reply:
Done.